### ISSUE TITLE
How to do a full DB load?

### ISSUE THREAD

**Issue by jplock** [trailofbits/cvedb#1]
Hi, thank you for creating this library.

I'm trying to follow the example but I'm having difficulty getting the CVE database to load:

```python
from cvedb.db import CVEdb

with CVEdb.open() as db:
  for cve in db.data():
    print(cve)
```

but it throws:

```python
NotImplementedError: Add support for CPE 23 URI node keys {'cpe_name'}
```

**Comment by ESultanik** [comment]
This should now be fixed in version v0.0.4, now available on PyPI.

**Comment by jplock** [comment]
Thank you @ESultanik !
